examples/CMakeLists.txt

```diff
@@ -12,15 +12,15 @@ if(NOT ignition-common2_FOUND)
   set(IGN_COMMON_VER ${ignition-common3_VERSION_MAJOR})
 endif()
```

Comment:
It's nice that the script caught this unusual use case (a library referring to sibling libs that aren't direct dependents). I think here we can just remove the logic for `common2`, `common3`, `common4`, `math5` and `math6`, because those will never be supported in combination with `plugin2`.

Reply:
I'm not sure how to do that :o
You mean on the release repo script?

Does this bump need to be reverted?

Reply:
Oh no, so what I'd do is just edit this file to remove the `if`s for the other versions.

The reason there are so many on `ign-plugin1` is that `ign-plugin1` was on multiple distros (from Acropolis to Fortress), so it could be combined with various `common` and `math` versions.

But for now, `ign-plugin2` will only be combined with the versions in Garden, i.e. `common5` and `math7`.

Reply:
Ohhh https://github.com/ignitionrobotics/ign-plugin/pull/74/files#diff-940e5356fa1137e646ea135a6f68cacbfad4fe7124c3a69163468f588acf9283

Reply:
Ohhh https://github.com/ignitionrobotics/ign-plugin/pull/74/files#diff-940e5356fa1137e646ea135a6f68cacbfad4fe7124c3a69163468f588acf9283